dashboard/views.py

```diff
@@ -142,6 +149,60 @@ def get_top_tracks(sp):
     return tracks
 
 
+def get_recent_tracks(sp):
+    recent_tracks = sp.current_user_recently_played(limit=10)
+    tracks = []
+
+    track_ids = [track["track"]["id"] for track in recent_tracks["items"]]
+    audio_features_list = sp.audio_features(track_ids)
+
+    existing_tracks = TrackVibe.objects.filter(track_id__in=track_ids)
+    existing_tracks_dict = {track.track_id: track for track in existing_tracks}
+
+    for track, audio_features in zip(recent_tracks["items"], audio_features_list):
+        track_vibe = existing_tracks_dict.get(
+            track["track"]["id"], TrackVibe(track_id=track["track"]["id"])
+        )
+        # Compute audio vibe for each track, doesn't matter if it was already in the database
+        track_vibe.track_audio_vibe = deduce_audio_vibe([audio_features])
+        track_vibe.save()
+        display_lyrics_vibe = ""
+        if track_vibe.track_lyrics_vibe is not None:
```

Comment:
Note: the lyrics_vibe for a track will display only if it has been computed in a previous scheduled asynchronous vibe calculation. This is motivated by: 1). Recent tracks update frequently, 2). Get lyrics vibe is slow and expensive, 3), we decided to schedule a lyrics vibe calculation every day.